src/components/avatar/_avatar.scss

```diff
@@ -26,27 +21,21 @@ $includeHtml: false !default;
     background: $avatarDefaultBackgroundColor;
 
     &__image {
-      position: absolute;
-      top: 0;
-      left: 0;
+      position: relative;
+      height: 100%;
       width: 100%;
-      background-color: $avatarBackgroundColor;
-    }
 
-    &:before {
-      position: relative;
-      top: 4px - $avatarMediumBorderWidth;
-      color: $avatarDefaultPersonColor;
-      font-size: $avatarMediumSize;
+      &--icon {
+        top: 3px;
```

Comment:
I will leave it like this for now lets review it while changing to male/female avatars.